src/capitalsFirst.js

```diff
@@ -20,8 +20,28 @@
  *
  * @returns {string} - string with uppercase words in front
  */
+function isUpperCased(word) {
+  return word && word[0] === word[0].toUpperCase() && /[A-ZА-Я]/.test(word[0]);
+}
+
+function isLowerCased(word) {
+  return word && word[0] === word[0].toLowerCase() && /[a-zа-я]/.test(word[0]);
+}
+
 function capitalsFirst(str) {
-  // write code here
+  let wordsLowerCase = [];
+  let wordsUpperCase = [];
+  let allWords = str.split(' ');
+  for (let word of allWords) {
+    if (isUpperCased(word)) {
+      wordsUpperCase.push(word);
+    } else if (isLowerCased(word)) {
+      wordsLowerCase.push(word);
+    }
+  }
+  let sortedWords = wordsUpperCase.concat(wordsLowerCase);
+  let result = sortedWords.join(' ');
+  return result;
```

Comment:
you can use spread operator here: `reutrn [...upperCaseWords, ...lowerCaseWords].join(' ');`